packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -220,17 +224,18 @@ export class FastSync extends (EventEmitter as {new (): InitialSyncEventEmitter}
    * Returns peers which has same finalized Checkpoint
    */
   private getInitialSyncPeers = async (): Promise<PeerId[]> => {
-    return this.network.getPeers().reduce((validPeers: PeerId[], peer: PeerId) => {
-      const rep = this.reps.getFromPeerId(peer);
-      if (
-        rep &&
-        rep.supportedProtocols.includes(Method.BeaconBlocksByRange) &&
-        rep.latestStatus &&
-        rep.latestStatus.finalizedEpoch >= this.targetCheckpoint!.epoch
-      ) {
-        validPeers.push(peer);
-      }
-      return validPeers;
-    }, [] as PeerId[]);
+    return this.network
```

Comment:
initially how can we have `peer.protocols`? curious how libp2p populate it without test requests

Reply:
It's using identify protocol:
https://github.com/libp2p/specs/tree/master/identify

If we start encountering that peer lies about it, we can add piece of code to remove it from protobook